cmd/root.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
```

Comment:
viper is a tool to manage configuration. It pairs well with cobra. I have not used it yet but decided to get it in already since I am sure that we will make use of it.